Formula/sile.rb

```diff
@@ -73,8 +73,8 @@ class Sile < Formula
   end
 
   resource "luaexpat" do
-    url "https://luarocks.org/manifests/tomasguisasola/luaexpat-1.3.3-1.src.rock"
-    sha256 "b55908fcd7df490a59aab25284460add8283f1c6b94ab584900fe3e49775172a"
+    url "https://luarocks.org/manifests/tomasguisasola/luaexpat-1.5.0-1.src.rock"
+    sha256 "e63ce827f930757d2977186af50971e0859d41ff904831056681037d66c5f03f"
```

Comment:
```suggestion
    url "https://luarocks.org/manifests/lunarmodules/luaexpat-1.5.1-1.src.rock"
    sha256 "eb7abc4cc39dd50eecb30896686bac84e66fd3c0774d0749b391e7ac5d222f45"
```

The one you had was 404.

Reply:
Yikes, I should have caught that. Thanks.